libs/core/langchain_core/runnables/base.py

```diff
@@ -2006,6 +2007,19 @@ async def _atransform_stream_with_config(
             if hasattr(iterator_, "aclose"):
                 await iterator_.aclose()
 
+    def as_tool(
+        self,
+        *,
+        name: Optional[str] = None,
+        description: Optional[str] = None,
+        arg_types: Optional[Dict[str, Type]] = None,
```

Comment:
not the biggest fan of this name. why not use `input_schema` or `paramaters` like the other major providers use?

Reply:
Ah ignore, I see you guys use `args_schema` in py where we use `schema` in js.